packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -56,7 +56,7 @@ var toArray = ((React.Children.toArray: any): toArrayType);
 var getStackAddendum = emptyFunction.thatReturns('');
 
 if (__DEV__) {
-  var validatePropertiesInDevelopment = function(type, props) {
+  let validatePropertiesInDevelopment = function(type, props) {
```

Comment:
Hm, seems to be a scope issue.
In this case I simple changed it and returns erros like `ReferenceError: validatePropertiesInDevelopment is not defined` on tests.

Reply:
Yep—you can define it outside the dev block.

Reply:
(but assign it inside)

Reply:
Maybe `let validatePropertiesInDevelopment = () => {};` would be the best default value, and then in the block you reassign.

Reply:
ok, I'll fix for `react-dom`, and see if forgot some var then. I'll add on this PR.

Reply:
It's fine for you?

Reply:
I think it's better I close this PR and send one with another branch name :P